backend/config/packages/test/monolog.yaml

```diff
@@ -15,5 +15,5 @@ monolog:
             channels: ['!event']
         nested:
             type: stream
-            path: '%kernel.logs_dir%/%kernel.environment%.log'
+            path: php://stderr
```

Comment:
You should not change this one, otherwise you end up mixing output from your test with logs from symfony, try it out.